test/wdl_2_0/apps_1421_dir_output_extras.json

```diff
@@ -19,7 +19,7 @@
     "apps_1421_task_01": {
       "runSpec": {
         "access": {
-          "project": "NONE",
+          "project": "VIEW",
```

Comment:
This setting is redundant since by default applets have VIEW access to the project
https://documentation.dnanexus.com/developer/api/running-analyses#example-chained-execution
https://documentation.dnanexus.com/developer/api/running-analyses#project-permissions

Reply:
Yep, sorry, just carryover from testing different combinations. I'll change if other changes needed. Otherwise don't want to trigger GHA again